modules-magic-drc/README.src.rst

```diff
@@ -0,0 +1,24 @@
+``skywater-pdk-actions`` - ``modules-magic-drc``
+====================================================
+
+This GitHub action runs Design Rule Checks on all GDS files inside the /cells directory.
+
+Written by `Donn <https://github.com/donn>`_ in collaboration with `Tim Edwards <https://github.com/rtimothyedwards>`_.
```

Comment:
This should just be an `AUTHORS` file.

Reply:
Was trying to stick to the structure by the other action, where there is no Authors file and the Authors are listed in the Readme.